Edit incremental search

yuka-matsuura opened the pull request:
# What
インクリメンタルサーチ後の処理追加

# Why
仕様通りの動作にするため

Commit:
Recover image file deleted accidentally

# What
誤って削除してしまった画像ファイルを復元

# Why
マイグレーションファイル新規作成時にNo such file or directory〜とエラーが出てしまったので、復元しています。

Commit:
Add incremental search

# What
インクリメンタルサーチの実装

# Why
グループ画面でのユーザー検索をインクリメンタルサーチで検索できる仕様だから

Commit:
Add incremental Search after

# What
インクリメンタルサーチ後の処理追加

# Why
仕様通りの動作にするため

Commit:
Change gitignore

# What
Github不要な画像をあげてしまったので、今後アップロードしないよう修正

メンターさんに「誤ってあげた画像ファイルが１つだけなので、画像ファイルを削除しなくても、gitignoreファイルの修正でいいです」といわれたの
でそのように対応しています。

# Why
不要な画像をあげないようにするため

Commit:
Change routing

# What
アクションの順番変更

# Why
可読性向上のため

Commit:
Fix group creation view

# What
グループ作成時、最初からグループメンバーにログインユーザが表示されるように修正

# Why
仕様通りに修正

MrUrakou commented on app/controllers/users_controller.rb:
```diff
@@ -1,5 +1,13 @@
 class UsersController < ApplicationController
 
+  def index
+    @users =User.where('name LIKE(?)', "%#{params[:keyword]}%")
```
=の後は半角スペース空けましょう。

MrUrakou commented on app/views/groups/_form.html.haml:
```diff
@@ -7,35 +7,29 @@
-        </div>
-        </div>
+      #chat-group-users
+        - @group.users.each do |user|
```
eachを仕様しないで実装しましょう。

MrUrakou commented on app/views/groups/_form.html.haml:
```diff
@@ -7,35 +7,29 @@
+      #chat-group-users
+        - @group.users.each do |user|
+          .chat-group-user.clearfix.js-chat-member#chat-group-user-8
+            %input{ name: "group[user_ids][]", type: "hidden", value: user.id}
```
current_userと紐付けましょう。

MrUrakou commented on app/views/groups/_form.html.haml:
```diff
@@ -7,35 +7,29 @@
+        - @group.users.each do |user|
+          .chat-group-user.clearfix.js-chat-member#chat-group-user-8
+            %input{ name: "group[user_ids][]", type: "hidden", value: user.id}
+            %p.chat-group-user__name= user.name
```
表示されているのはcurrent_userです。

Commit:
Fix index action and group form view

# What
指摘箇所の修正

レビュアーさんとお話しさせていただきまして、指摘箇所は下記２点のみ対応しました。
・users_controller.rbの@users定義箇所
・views/groups/_form.html.hamlのログインユーザの表示箇所

yuka-matsuura commented:
ご確認ありがとうございます。
指摘箇所のうちcontrollerの@usersの定義箇所とviewのログインユーザの表示の部分のみ修正いたしました。
※レビュアーさんとお話しした上で対応しております。
お手数をおかけしますが、再度ご確認のほどよろしくおねがいたします。

yuka-matsuura commented:
LGTMいただきましたので、masterブランチにマージさせていただきます。
ご確認ありがとうございました。
